script/test.py

```diff
@@ -242,10 +244,15 @@ def run_test(path):
       print('      ' + color.PINK + fail + color.DEFAULT)
     print('')
 
+def run_example(path):
+  return run_test(path, example=True)
 
 for dir in ['core', 'io', 'language', 'limit', 'meta']:
   walk(join(WREN_DIR, 'test', dir), run_test)
 
+walk(join(WREN_DIR, 'example'), run_example)
+walk(join(WREN_DIR, 'example', 'import-module'), run_example)
```

Comment:
Hmm. I didn't think it did given this line. https://github.com/munificent/wren/pull/268/files#diff-1d8e229374da562b542ae6b3bbeccc1bR250

I'll check and update as needed.